web/vital_records/views/common.py

```diff
@@ -86,7 +86,8 @@ class TypeView(EligibilityMixin, ValidateRequestIdMixin, UpdateView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["page_title"] = "Replacement records"
-        context["previous_route"] = Routes.request_start
+        context["previous_route"] = Routes.app_route(Routes.request_start)
+        context["previous_url"] = reverse(context["previous_route"])
```

Comment:
This is nitpicky, but... since the template no longer uses `previous_route`, we don't need it to be in the `context` dictionary, right?

We could have it as just a local variable, like:
```suggestion
        previous_route = Routes.app_route(Routes.request_start)
        context["previous_url"] = reverse(previous_route)
```

Same comment for all the other views and the mixin where we say `context["previous_url"] = reverse(...`